test/operators/pagerduty.go

```diff
@@ -0,0 +1,16 @@
+package operators
+
+import (
+	"github.com/onsi/ginkgo"
+	"github.com/openshift/osde2e/pkg/helper"
+)
+
+var _ = ginkgo.Describe("[OSD] Pagerduty Operator", func() {
```

Comment:
Could you add the string `[Suite: operators]` to the beginning of this Ginkgo.describe string? This is a brand new thing. If you rebase on master you'll see examples in the other operator tests.

Reply:
Cool will do that now! thanks